libs/components/layout/src/lib/modules/description-list/description-list-term.component.ts

```diff
@@ -21,4 +24,6 @@ export class SkyDescriptionListTermComponent {
     static: true,
   })
   public templateRef: TemplateRef<unknown> | undefined;
+
+  protected readonly content = inject(SkyDescriptionListContentComponent);
```

Comment:
This needs to be an optional injection since it's not guaranteed that the consumer will wrap this component within a content component.